src/Microsoft.Security.Utilities.Core/Base62.cs

```diff
@@ -118,30 +118,30 @@ public static T FromBase62<T>(this string base62, bool inverted = false)
             switch (Type.GetTypeCode(typeof(T)))
             {
                 case TypeCode.String:
-                return (T)Convert.ChangeType(Encoding.UTF8.GetString(array), typeof(T), CultureInfo.InvariantCulture);
+                    return (T)Convert.ChangeType(Encoding.UTF8.GetString(array), typeof(T), CultureInfo.InvariantCulture);
```

Comment:
There's a whole bunch of reformatting like this coming in with the last commit. Can you revert that commit and just fix the extra newline?

I will do #153 soon and fix formatting like this in a separate PR before turning on the enforcement.